images/macos/toolsets/toolset-13.json

```diff
@@ -3,17 +3,15 @@
         "default": "14.3.1",
         "x64": {
             "versions": [
-                { "link": "15.0.1", "version": "15.0.1-Release.Candidate+15A507"},
-                { "link": "15.0", "version": "15.0.0+15A240d"},
+                { "link": "15.0.1", "version": "15.0.1+15A507", "symlinks": ["15.0", "15.0.0"]},
```

Comment:
@sergei-pyshnoi you do not need `15.0.0` here

Reply:
removed